Jolt/Physics/PhysicsSystem.h

```diff
@@ -28,7 +28,8 @@ class PhysicsSystem : public NonCopyable
 	JPH_OVERRIDE_NEW_DELETE
 
 	/// Constructor / Destructor
-								PhysicsSystem()												: mContactManager(mPhysicsSettings) { }
+								PhysicsSystem()												: mPhysicsSettings(), mContactManager(mPhysicsSettings) { }
```

Comment:
gcc triggerred a warning about mPhysicsSettings not being initialized, so I had to do it like this. Adding this in the constructor forced me to change later the order of the member variables because of a trigger of -Wreorder.